test/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "rules": {
+    "no-invalid-this": 0,
+    "no-console": 0,
```

Comment:
I think that unit tests should not be allowed to use the console, but should instead use `should` error system, because of the way `should` handle errors: during tests it displays only a `test failed` line, and after running every tests it can, it displays all necessary informations on the console, including a stack trace.
So printing an error on the console would display it with the single `test failed` line, and it would easily go unnoticed if one use only the full error display at the end of the test sequence.